webapp/src/component/security/LoginView.tsx

```diff
@@ -26,16 +33,35 @@ import {
 
 interface LoginProps {}
 
+type LoginRequestDto = components['schemas']['LoginRequest'];
+
 const globalActions = container.resolve(GlobalActions);
 const securityServiceIns = container.resolve(SecurityService);
-// noinspection JSUnusedLocalSymbols
-export const LoginView: FunctionComponent<LoginProps> = (props) => {
+
+type Credentials = { username: string; password: string };
+type LoginViewCredentialsProps = {
+  credentialsRef: RefObject<Credentials>;
+  onMfaEnabled: () => void;
+};
+type LoginViewTotpProps = {
+  credentialsRef: RefObject<Credentials>;
+  onMfaCancel: () => void;
+};
+
+export function LoginViewCredentials(props: LoginViewCredentialsProps) {
```

Comment:
Plz make these separate files, it's easier to understand when it's one file - one component